src/components/ToastNotification/ToastNotification.stories.tsx

```diff
@@ -94,7 +94,7 @@ const ToastNotificationManager = (args: Args) => {
         Trigger A Toast Notification
       </Button>
       <div
-        className="dur absolute bottom-0 left-0 flex flex-col gap-size-2"
+        className="dur gap-size-2 absolute bottom-0 left-0 flex flex-col"
```

Comment:
some of these with `-size-` slide around b/c we now need to update it to things like `gap-spacing-size-2`. Also for this one line ... that `dur` is suspicious. maybe it should be deleted while we are here?